### DISCUSSION TITLE
Favorite button next to theme in the bottom right

### DISCUSSION THREAD

**Discussion by MaxACartwright** [monkeytypegame/monkeytype#2863] [Ideas]
Just something quick and dirty, a small change really.

If you could add a favorite star right next to the current theme in the bottom right (see below), that would be cool.
![image](https://user-images.githubusercontent.com/51039735/164945720-2e691bd6-c540-4bec-95b6-75c27a0c2f71.png)
Like right next to the theme just put a little star. It would really help out for the people who shuffle themes and are looking for some good ideas.
Bonus points if you can create a notification at the top right saying something along the lines of "Favorite added"
Thanks!
- Maximus

**Comment by aNOOBisTheGod** [comment]
I think you can actually save current settings preset and name it "FavoriteTheme" so this feature is kinda useless. But it still can exist

(and btw there is actually "favorite" button in settings

**Reply by MaxACartwright** [reply]
I'm aware of the favorite button in settings, I just think it would be a timesaver to _instantly_ favorite the theme instead of having to go into the settings and do it.

**Reply by aNOOBisTheGod** [reply]
I've taught about this... and it's quite useless....

users quite rarely adds theme to favorite
